src/box/lua/config/init.lua

```diff
@@ -192,8 +192,11 @@ function methods._collect(self, opts)
     end
```

Comment:
Nit: I think it worth to write a changelog entry, because now we have a baseline: 3.0.0-alpha1.

Reply:
Done